pennylane_lightning/lightning_qubit/lightning_qubit.toml

```
@@ -118,7 +118,7 @@ runtime_code_generation = false
 # does not support mid-circuit measurements.
 supported_mcm_methods = [ "one-shot" ]
 # Whether the device supports dynamic qubit allocation/deallocation.
-dynamic_qubit_management = false
+dynamic_qubit_management = true
```

Hmm, this means a different thing! dynamic allocation here refers to partial allocation and release of qubits at runtime.
 ```suggestion
dynamic_qubit_management = false
```

---

I see...